src/lib/pages/code-details/components/code-info/CodeInfoSection.tsx

```diff
@@ -194,34 +190,28 @@ export const CodeInfoSection = ({
         </LabelText>
         {!isMobile && (
           <LabelText label="JSON Schema">
-            {isCodeHashLoading ? (
+            {!code ? (
```

Comment:
Let's eliminate the loading render state since `isCodeHashLoading` does not exist and the hash is already available in the code object.